packages/@aws-cdk/aws-kinesis/package.json

```diff
@@ -60,14 +60,30 @@
     "url": "https://aws.amazon.com",
     "organization": true
   },
+  "jest": {
```

Comment:
Do we really need to duplicate all this config everywhere? Isn't there a way to use a default config? @RomainMuller ?

Reply:
I believe we don't need this anymore, I know some modules simply have `"jest": {}` @nija-at?

Reply:
@iliapolo @eladb - you're right - we did centralize this - updated!

I think we still have a whole bunch of modules that have jest keys configured, as only about ~15 modules use this style. I'll clean em up in a separate PR

Reply:
https://github.com/aws/aws-cdk/pull/6869